src/Uno.UI/Media/PathStreamGeometryContext.cs

```diff
@@ -133,11 +133,21 @@ public override void ArcTo(Point point, Size size, double rotationAngle, bool is
 			);
 
 #elif __MACOS__
-			bezierPath.AppendPathWithArc(center,
+//Ugly workaround. check if all vars are defined
+			if (!double.IsNaN(radius) && !double.IsNaN(startAngle) && !double.IsNaN(endAngle)) {
```

Comment:
What case did you see causes this issue ?

Reply:
On the initialization process, a NaN would be passed. This caused the arc to not show.
If you resized the window, the arc would be displayed again